components/server/ee/src/workspace/workspace-factory.ts

```diff
@@ -236,27 +236,31 @@ export class WorkspaceFactoryEE extends WorkspaceFactory {
         }
     }
 
-    private async isGoodBaseforIncrementalPrebuild(
+    public async isGoodBaseforIncrementalPrebuild(
```

Comment:
I wonder if we can extract this method from WorkspaceFactory as well.
Maybe put it into a PrebuildFactory (or similar), together with the method from GitpodServerImpl above?
Just a suggestions, please decide yourself.